### DISCUSSION TITLE
希望增加网页投稿可以分p投稿

### DISCUSSION THREAD

**Discussion by pcbwsxkm** [the1812/Bilibili-Evolved#4709] [功能建议 / Ideas]
应该是可以分p投稿的，用客户端分p投稿后网页才出现添加视频这个按钮

**Comment by Geeyun-JY3** [comment]
@the1812 我只记得明确拒绝过`夜间模式`适配创作中心： #1922，但印象中与投稿、创作中心相关的需求其实都是拒绝实现的（全站生效的功能也都是尽量排除相关页面，尽可能避免影响相关页面），或者可以也写进讨论区置顶的[不会做的功能](https://github.com/the1812/Bilibili-Evolved/discussions/1301#discussioncomment-8787140)？

**Reply by the1812** [reply]
可以加一下 (虽然这条是因为命中了 `恢复任何旧版本页面`

**Comment by 1779587460** [comment]
感觉挺重要的，但被拒绝了T-T

**Reply by Tinhone** [reply]
急切有这个需求的话也许可以尝试一下 [biliup-rs](https://github.com/biliup/biliup-rs)
